qiskit/circuit/library/basis_change/qft.py

```diff
@@ -315,8 +315,10 @@ def __init__(
         """
         super().__init__(name="qft", num_qubits=num_qubits, params=[])
 
-    def __array__(self, dtype=complex):
+    def __array__(self, dtype=complex, copy=None):
         """Return a numpy array for the QFTGate."""
+        if copy is False:
+            raise ValueError("unable to avoid copy while creating an array as requested")
```

Comment:
Tiny tiny suggestion:

```suggestion
            raise ValueError("Unable to avoid copy while creating an array as requested")
```

Reply:
Thanks @ElePT! I agree that the capitalized "U" looks better, but all the other 30+ identical message use lower-case "u".

Reply:
hah, tunnel vision. Let's keep it consistent then.